api/test/tests/controllers/topic.js

```diff
@@ -237,4 +238,35 @@ describe( 'api/lib/controllers/topic', () => {
     });
   });
 
+  describe( '#getTakeaways', () => {
+    const path = '/takeaway';
+
+    it( 'should get topic\'s takeaways', async() => {
+      const topic = topicFaker();
+
+      const res = await client.post( '/topic', topic );
```

Comment:
Instead of sending a post request can you just insert it into the db. The issue with using a post request is that it starts to rely on our code for post requests of topics. As a result, when the topic create controllers breaks, this test will break as well which can be confusing to resolve.